docs/playbook-reference/defining-playbooks/external-playbook-repositories.rst

```diff
@@ -109,6 +116,20 @@ The ``key`` parameter must contain a ``base64`` encoded deployment key with ``re
         ewfrcfsfvC1rZXktdjEAAAAABG5vb.....
         -----END OPENSSH PRIVATE KEY-----
 
+Loading Actions from an external Python Package
+---------------------------------------------------
+
+For external Python packages, just specify an URL starting with http(s), and ending with
+either .tar, .tar.gz, .tgz, .tar.bz2, or .tbz2.
+
+.. code-block:: yaml
+
+    playbookRepos:
+      web_playbooks:
+        url: "https://my-domain.com/bla/web-playbooks.tgz"
+        http_headers: # optional, may be used for auth
```

Comment:
i think we should either allow this to be used for all the urls or specify here that its only for zip and use it in the validator

Reply:
It's explicitly requested (see https://robusta.youtrack.cloud/issue/MAIN-1815/load-actions-repo-from-https-get-url) that this only applies to external Python packages (tgz etc) loaded via the mechanism implemented here.

Reply:
I made it more clear in the docs.